source/platform/file_formats/ncch.hpp

```diff
@@ -466,14 +497,45 @@ struct NCSDHeader {
             (MediaUnit32, size)
         );
     };
+    
+    struct CartHeader {
+        BOOST_HANA_DEFINE_STRUCT(CartHeader,
+            (std::array<uint8_t, 32>, exheader_sha256),
+            (uint32_t, additional_header_size),
```

Comment:
If this refers to NCSDExtendedHeader, could you add a comment like this?

```suggestion
            (uint32_t, additional_header_size), // see NCSDExtendedHeader
```